filesystem/azure.go

```diff
@@ -0,0 +1,196 @@
+package filesystem
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"github.com/MSOpenTech/azure-sdk-for-go/storage"
+	"io"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+)
+
+type AzureClient struct {
```

Comment:
The program is implement for filesystem's interfaces. 
Hdfs.go is in the same package, thus I think it should name AzureClient to distinguish.